src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -438,24 +437,26 @@ private static void logCommand(List<String> command) {
     logger.info("submitting command: " + WHITESPACE_JOINER.join(command));
   }
 
-  public Path getSdkPath() {
-    return sdkPath;
+  /** @return the directory containing the Cloud SDK installation backing this instance */
+  public Path getPath() {
+    return cloudSdkPath;
   }
 
   private Path getGCloudPath() {
     String gcloud = GCLOUD;
     if (IS_WINDOWS) {
       gcloud += ".cmd";
     }
-    return getSdkPath().resolve(gcloud);
+    return getPath().resolve(gcloud);
   }
 
   private Path getDevAppServerPath() {
-    return getSdkPath().resolve(DEV_APPSERVER_PY);
+    return getPath().resolve(DEV_APPSERVER_PY);
   }
 
-  public Path getJavaAppEngineSdkPath() {
-    return getSdkPath().resolve(JAVA_APPENGINE_SDK_PATH);
+  /** @return the directory containing JAR files bundled with the Cloud SDK */
```

Comment:
I don't think this rename is helpful as there are a number of directories in the SDK that contain .jar files.

Reply:
renamed